docs/integration/kubernetes.md

```diff
@@ -0,0 +1,257 @@
+Kubernetes is a portable, extensible, open-source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. To start monitoring Kubernetes with Sematext, you only need to install a tiny agent that adds basically no CPU or memory overhead.
+
+
+## Monitoring Kubernetes with Sematext
+
+Sematext Monitoring will give you detailed insights into your cluster’s health, performance metrics, resource counts amongst other important metrics. Speaking of metrics, check out [this ](https://sematext.com/docs/agents/sematext-agent/kubernetes/metrics/)page for a summarized list of the key metrics you can follow with Sematext as well as a short explanation for each one of them.
+
+
+
+
+![alt_text](https://sematext.com/wp-content/uploads/2020/03/Kubernetes-Pod-Overview.png "Sematext Monitoring")
+
+
+
+### Helm Chart
+
+To get started with monitoring Kubernetes with Sematext you’ll have to install the Sematext Agent and the easiest way to do it is with a Helm chart. It’s available in the official charts repo and it will install to all the nodes in your cluster. To install it you’ll need to run the following command:
+
+
+
+
+```
+helm install --name sematext-agent \
+  --set containerToken=<YOUR_CONTAINER_TOKEN> \
+  --set infraToken=<YOUR_INFRA_TOKEN> \
+  --set logsToken=<YOUR_LOGS_TOKEN> \
+  --set region=<"US" or "EU"> \
+  stable/sematext-agent
+```
+
+
+
+Check out [github ](https://github.com/helm/charts/blob/master/stable/sematext-agent/README.md)for more details.
```

Comment:
When you have )for, meaning no space after ), does the space still get added? (don't have the ability to preview this, so asking)